x-pack/plugins/apm/server/lib/service_map/get_service_anomalies.ts

```diff
@@ -130,13 +143,15 @@ function transformResponseToServiceAnomalies(
     response.aggregations?.services.buckets ?? []
   ).reduce(
     (statsByServiceName, { key: serviceName, top_score: topScoreAgg }) => {
+      const mlResult = topScoreAgg.hits.hits[0]._source;
```

Comment:
You had this as a maybe previously? Was that not the case?
(I like it better now for sure)

Reply:
It's always there, because we don't use min_doc_count. So the `maybe` here was too defensive anyway.

Reply:
Great 👍